pages/draft-05/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Draft-05'
+Published: '21-April-2017'
+type: docs
+authors: ['Austin Wright', 'Henry Andrews', 'Ben Hutton', 'Greg Dennis']
+Metaschema: 'https://json-schema.org/draft/2020-12/schema'
+Specification: 'https://json-schema.org/draft/2020-12/json-schema-core.html'
```

Comment:
Those links are not correct.

Reply:
For Draft 5, I used the metaschema of Draft 4.